src/main/kotlin/org/opensearch/commons/alerting/action/DeleteWorkflowRequest.kt

```diff
@@ -0,0 +1,39 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.action.ActionRequest
+import org.opensearch.action.ActionRequestValidationException
+import org.opensearch.action.support.WriteRequest
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import java.io.IOException
+
+class DeleteWorkflowRequest : ActionRequest {
+
+    val workflowId: String
+    val deleteDelegateMonitors: Boolean?
+    val refreshPolicy: WriteRequest.RefreshPolicy
```

Comment:
remove refresh policy 
we shoudl alway set it as IMMEDIATE and not take input from user